controllers/reconcile.go

```diff
@@ -401,7 +404,10 @@ func createConfigConfigMap(ctx context.Context, pClient client.Client, frontend
 
 	for _, frontend := range frontendList.Items {
 		if frontend.Spec.Module != nil {
-			modName := frontend.GetName()
+			// module names in fed-modules.json must be camelCase
+			// K8s does not allow camelCase names, only
+			// whatever-this-case-is, so we convert.
+			modName := strcase.ToLowerCamel(frontend.GetName())
```

Comment:
We will use our own `ToLowerCamel` function, this was proving our fix for this specific case.